### ISSUE TITLE
Bug:Assimp cannot correctly parse the normals of morphTaget from the fbx file

### ISSUE THREAD

**Issue by Brave-Pony** [assimp/assimp#5877]
**Describe the bug**
I have a problem where assimp cannot correctly parse the normals of morphTaget from the fbx file, the parsing result is always similar to the normals of basicShape. Why is this?

**To Reproduce**
1. Open blender(v3.6.8).
2. Create a plane object and make two simple morph targets.
3. export the plane as fbx file by the default tool of blender.
4. run assimp to importer the fbx file, and print the normals of morphTarget.

**Expected behavior**
hope to obtain the normals data of the morphoTarget stored in fbx file.

**Screenshots**
none

**Platform (please complete the following information):**
 - windows 10
 - assimp 5.4.3 or 5.2.5

**Additional context**
none

**Comment by Brave-Pony** [comment]
> Please attach sample 3D model file with morph targets

[plane.zip](https://github.com/user-attachments/files/17783058/plane.zip)
This one.

**Comment by tellypresence** [comment]
I get the same result, `plane.fbx` blendshape normals don't reflect actual plane orientation during morph, and remain the same as for the base mesh.

(For comparison looked at AnimatedMorphCube from the Khronos gltf model repo; this model has correct normals when morphing between blendshapes)

<table>
<tr><th>Model</th><th>Debug normals Screenshot<br>X=Red, Y=Green, Z=Blue</th><th>Notes</th></tr>
<tr><td><tt>plane.fbx</tt></td><td><img src="https://github.com/user-attachments/assets/a3d8d3a8-f40b-4f08-983b-0bb6d9a93964" alt="plane_fbx_normals_incorrect" width=90 ></td><td>Blendshape normals unchanged from base mesh; the normal of the half folding upwards points more along X axis and so should take on increasingly more reddish tint</td></tr>
<tr><td><tt>AnimatedMorphCube.gltf</tt></td><td><img src="https://github.com/user-attachments/assets/ed030e89-bf34-465b-943a-9cfbd4350e18" alt="AnimatedMorphCube_gltf_normals_correct" width=90 ></td><td>Blendshape normals update correctly</td></tr>
</table>

**Comment by Brave-Pony** [comment]
> I get the same result, `plane.fbx` blendshape normals don't reflect actual plane orientation during morph, and remain the same as for the base mesh.
> 
> (For comparison looked at AnimatedMorphCube from the Khronos gltf model repo; this model has correct normals when morphing between blendshapes)
> 
> Model	Debug normals Screenshot
> X=Red, Y=Green, Z=Blue	Notes
> plane.fbx	![plane_fbx_normals_incorrect](https://private-user-images.githubusercontent.com/26958898/386919528-a3d8d3a8-f40b-4f08-983b-0bb6d9a93964.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE4OTQzNTQsIm5iZiI6MTczMTg5NDA1NCwicGF0aCI6Ii8yNjk1ODg5OC8zODY5MTk1MjgtYTNkOGQzYTgtZjQwYi00ZjA4LTk4M2ItMGJiNmQ5YTkzOTY0LmdpZj9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTE4VDAxNDA1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWZkNzIzZjA2OWU2NzNiY2Q1YjM5NTgyNDNmZGY4ZWFmYjA3MjBiMWUxZGU4NGYzN2ZmNWJiNTc0YjI1OTk1MGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.qICvE11KvSW7kYgi22e0HFX5nYXgEeIH1cLYZYY2gLI)	Blendshape normals unchanged from base mesh; the normal of the half folding upwards points more along X axis and so should take on increasingly more reddish tint
> AnimatedMorphCube.gltf	![AnimatedMorphCube_gltf_normals_correct](https://private-user-images.githubusercontent.com/26958898/386919503-ed030e89-bf34-465b-943a-9cfbd4350e18.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE4OTQzNTQsIm5iZiI6MTczMTg5NDA1NCwicGF0aCI6Ii8yNjk1ODg5OC8zODY5MTk1MDMtZWQwMzBlODktYmYzNC00NjViLTk0M2EtOWNmYmQ0MzUwZTE4LmdpZj9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTE4VDAxNDA1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWNlZTk3ODZjYjVlMWRjMGE5YzZjNGE0NThhMmZjODA2MGQwZjNkNDE1OTI2ZmNkZjAzMmE2ZjVmNjdlMmNkZmUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.1DK-I_jm2F2QekwVW7QFC8Im2wx7kVn2AXNER6c-4dk)	Blendshape normals update correctly

Yes, I understand two situations. I have also tested gltf and fbx before. The gltf file format can be loaded correctly.

But in our project, we mainly use FBX format files. We do not want to convert from fbx to gltf format.

I have been using other tools to check the content of plane.fbx (directly using fbxsdk or some fbx format converters), and I am sure that the file contains the correct normal data for blendShape. Why can't assimp load it?

**Comment by tellypresence** [comment]
FBX import issues tracked via epic #6156
